pages/checkin.tsx

```diff
@@ -41,8 +45,19 @@ const CheckinResult = ({ result, resetResults }) => {
       });
 
       if (!assignRequest.error) {
-        alert("Successfully assigned hacker QR");
-        resetResults();
+        const checkinRequest = await liveDispatchFetch({
+          actionId: "checkin",
+          qrCodeId: qrInputValue
+        });
```

Comment:
We have to use async/await due to fetcher's return type being async func?

Reply:
yup